modules/ROOT/pages/design-edit-mulitple-tabs.adoc

```diff
@@ -0,0 +1,40 @@
+= Edit in Multiple Tabs in the Text Editor
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can have multiple files open at a time in API Designer's text editor.
```

Comment:
Consider "at a time" > "simultaneously" for international audience.

Reply:
Should we say WHY the audience wants or needs to do this?
Are there any consequences to be aware of in advance?

Reply:
Should this intro. also mention navigation and closing tabs, since they are also subtopics?